ios/ScreenSrollableContentWrapper.h

```diff
@@ -0,0 +1,13 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef void (^LayoutCallback)(CGRect frame);
+
+@interface ScreenSrollableContentWrapper : UIScrollView
```

Comment:
```suggestion
@interface ScreenScrollableContentWrapper : UIScrollView
```

Reply:
Removed this file, as different solution was found.